flexget/plugins/sites/rlsbb.py

```diff
@@ -0,0 +1,174 @@
+    def url_rewrite(self, task, entry):
+        soup = self._get_soup(task, entry['url'])
+
+        # grab links from the main post:
+        link_elements = []
+        log.debug('Searching %s for a tags where the text matches one of: %s',
+                  entry['url'], str(self.config.get('link_text_re')))
+        for regexp in self.config.get('link_text_re'):
+            link_elements.extend(soup.find_all('a', string=re.compile(regexp)))
+        log.debug('Original urls: %s', str(entry['urls']))
+        if entry['urls']:
+            urls = list(entry['urls'])
+        else:
+            urls = []
+        log.debug('Found link elements: %s', str(link_elements))
+        for element in link_elements:
+            if re.search('nfo1.rlsbb.(ru|com)', element['href']):
+                # grab multipart links
+                urls.extend(self.grab_multilinks(task, element['href']))
+            else:
+                urls.append(element['href'])
+
+        # grab links from comments
+        regexps = self.config.get('filehosters_re', [])
+        if self.config.get('parse_comments'):
+            comments = soup.find_all('div', id=re.compile("commentbody"))
+            log.debug('Comment parsing enabled: found %d comments.', len(comments))
+            if comments and not regexps:
+                log.warn('You have enabled comment parsing but you did not define any filehoster_re filter. You may get a lot of unwanted and potentially dangerous links from the comments.')
+            for comment in comments:
+                links = comment.find_all('a')
+                for link in links:
+                    urls.append(link['href'])
+
+        # filter urls:
+        filtered_urls = []
+        for i, url in enumerate(urls):
+            urls[i] = normalize_unicode(url)
+            for regexp in regexps:
+                if re.search(regexp, urls[i]):
+                    filtered_urls.append(urls[i])
+                    log.debug('Url: "%s" matched filehoster filter: %s', urls[i], regexp)
+                    break
+            else:
+                if regexps:
+                    log.debug(
+                        'Url: "%s" was discarded because it does not match any of the given filehoster filters: %s', urls[i], str(regexps))
+        if regexps:
+            log.debug('Using filehosters_re filters: %s', str(regexps))
+            urls = filtered_urls
+        else:
+            log.debug('No filehoster filters configured, using all found links.')
+        num_links = len(urls)
+        log.debug('Original urls: %s', str(entry['urls']))
+        log.verbose('Found %d links at %s.', num_links, entry['url'])
+        if num_links:
+            entry['urls'] = urls
+            entry['url'] = urls[0]
+        else:
+            raise UrlRewritingError('No useable links found at %s' % entry['url'])
+
+
+@event('plugin.register')
+def register_plugin():
+    plugin.register(UrlRewriteRlsbb, 'rlsbb', interfaces=[
```

Comment:
I guess this was a relic from before I set pep8 line width to 120. Fixed it now.